src/main/java/org/folio/circulation/rules/Text2Drools.java

```diff
@@ -345,6 +345,8 @@ private String policyMatchString(PolicyContext policy) {
       policyTypeName = "request";
     } else if (policy.POLICY_TYPE().toString().equals("n")) {
       policyTypeName = "notice";
+    } else if (policy.POLICY_TYPE().toString().equals("o")) {
+      policyTypeName = "overdue";
```

Comment:
Can you make the method `static`?

Reply:
@julianladisch Could you please provide some arguments regarding this method signature change?

Reply:
The method is private so the signature change cannot break any compatibility.
Adding `static` documents that there is no dependency on the class variables.
Unit testing is more easy when the method is static because no instance of the class needs to be created.
See also https://rules.sonarsource.com/java/RSPEC-2325

Reply:
Ok. I will do it! Thank you for your explanation.